src/lib/bakeryjs/Box.ts

```diff
@@ -5,19 +5,14 @@ import {
 	BoxMeta,
 	OnCleanCallback,
 } from './BoxI';
-import {
-	DataMessage,
-	isData,
-	isSentinel,
-	Message,
-	MessageData,
-	SentinelMessage,
-} from './Message';
+import {DataMessage, isData, isSentinel, Message, MessageData} from './Message';
 import {PriorityQueueI} from './queue/PriorityQueueI';
 import VError from 'verror';
 import {ServiceProvider} from './ServiceProvider';
 import {AssertionError} from 'assert';
 import ajv from 'ajv';
+import {default as BoxEvents} from './BoxEvents';
```

Comment:
S ohledem na konzistenci (a [dobrou praxi](https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad)) bych to neexportoval jako `default`.